common/v2/features/Toasts/constants.ts

```diff
@@ -45,5 +46,14 @@ export const toastConfigs: ToastConfigsProps = {
     message: () => 'All systems good to go.',
     type: ToastType.SUCCESS,
     position: 'top-left'
+  },
+  [ToastTemplates.walletConnectReject]: {
+    header: 'Connection request rejected',
```

Comment:
Use translation strings

Reply:
the toast is out of service so I removed it.